CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 Interested in contributing? We'd love your help!
 
+If you run into problems, or find something confusing, please share it with us in [this discussion](https://github.com/pikapkg/snowpack/discussions/958). A great experience for (new) contributors is very important to us.
```

Comment:
I wonder if there is a better way to frame this. Maybe

```suggestion
We appreciate you trying to contribute to Snowpack. If something is confusing or unnecessary complex, please let us know in [this discussion](https://github.com/pikapkg/snowpack/discussions/958). Lowering the bar to (new) contributors such as yourself is very important to us.
```

I'm open to ideas :D

Reply:
I think as written is perfect, tbh. 

Might do better at the bottom though, as a kind of "can't find what you're looking for?"

nit: `If you run into problems or find something confusing, please...` (no comma needed)

nit: `A great experience for new contributors is very important to us!` (no need for parens, and a little extra "!" at the end)

Reply:
```suggestion
If you run into problems or find something confusing, please share it with us in [this discussion](https://github.com/pikapkg/snowpack/discussions/958). A great experience for new contributors is very important to us!
```